### PR TITLE
Add a payload verifier for DDR3 and multiple test payloads.

### PR DESCRIPTION
**Pull request by yoongu**

### CODE REVIEW

File: scripts/ddr3lib.py [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    if nop.opcode != Opcode.NOP:
+      return False
+    if not (0 < nop.timeslice < (1 << Instr.NopInstr.Bits.TIMESLICE)):
+      return False
+    return True
+  if instr.HasField('jmp'):
+    jmp = instr.jmp
+    if jmp.opcode != Opcode.JMP:
+      return False
+    if not (0 < jmp.offset < (1 << Instr.JmpInstr.Bits.OFFSET)):
```
Comment (sqazi) [inline scripts/ddr3lib.py:36]: Is it worth verifying the jump target being a valid instruction?

Comment (yoongu) [reply scripts/ddr3lib.py:36]: Yup, we do that in main(). Here we are just checking for SIGILLs.

Comment (yoongu) [reply scripts/ddr3lib.py:35]: On second thought, that's a good idea. Done in the follow-up commit.

**Comment by yoongu** [comment]
The second commit reduces the verification time of one million ACT/PRE loop from ~24 seconds to ~14 seconds. Still incredibly slow compared to the C++ version, but I think it might be good enough for now.

**Pull request merged**